### PR TITLE
Change address on access request dialog

### PR DESCRIPTION
**Pull request by KillariDev**
- Fixes https://github.com/DarkFlorist/TheInterceptor/issues/24
- Also redesigns request dialog to align the new style

Adds address change button on simulation mode:
image

And a refresh button on signer mode:
image

### CODE REVIEW

**Comment by MicahZoltu** [comment]
I would want the list of associable addresses to be expanded by default.  I can't imagine a situation where I *wouldn't* want to see the list.

**Review by MicahZoltu — APPROVED** [review]

**Comment by KillariDev** [comment]
> I would want the list of associable addresses to be expanded by default. I can't imagine a situation where I _wouldn't_ want to see the list.

its expanded by default, unless its empty, then its closed

File: extension/app/ts/background/windows/interceptorAccess.ts [hunk 1]
```diff
@@ -1,161 +1,255 @@
...
+			const associatedAddresses = getAssociatedAddresses(window.interceptor.settings, origin, newActiveAddressAddressInfo)
+			pendingInterceptorAccess = {
+				future: new Future<InterceptorAccessOptions>(),
+				origin: origin,
+				requestAccessToAddress: newActiveAddress,
+			}
+			await sendPopupMessageToOpenWindows({
+				method: 'popup_interceptorAccessDialog',
+				data: {
+					title: 'TODO add title',
```
[the remark this answers is not in the document]
Comment (KillariDev) [reply extension/app/ts/background/windows/interceptorAccess.ts:241]: yeah, thats in next pr

File: extension/app/ts/components/pages/InterceptorAccess.tsx [hunk 2]
```diff
@@ -1,20 +1,126 @@
...
+import { getSignerName } from '../subcomponents/signers.js'
+
+function AssociatedTogether({ associatedAddresses, renameAddressCallBack }: { associatedAddresses: readonly AddressInfoEntry[], renameAddressCallBack: RenameAddressCallBack } ) {
+	const [showLogs, setShowLogs] = useState<boolean>(associatedAddresses.length > 1)
+
+	return <>
+		<div class = 'card' style = 'margin-top: 10px; margin-bottom: 10px'>
+			<header class = 'card-header noselect' style = 'cursor: pointer; height: 30px;' onClick = { () => setShowLogs((prevValue) => !prevValue) }>
+				<p class = 'card-header-title' style = 'font-weight: unset; font-size: 0.8em;'>
+					{ associatedAddresses.length <= 1 ? 'The website cannot associate any addresses with each other' : <>
```
[the remark this answers is not in the document]
Comment (KillariDev) [reply extension/app/ts/components/pages/InterceptorAccess.tsx:20]: you can remove Vitalik if you want

[the remark this answers is not in the document]
Comment (KillariDev) [reply extension/app/ts/components/pages/InterceptorAccess.tsx:20]: and also, if you are using vitalik, we are not associating any addresses together. But maybe we should show here that the website can see vitalik 🤔

**Pull request merged**